### PR TITLE
optim: made loading docs more elaborate

### PR DESCRIPTION
**Pull request by codebender828**
Hello, Nuxt team. ❤️🎉

Thanks for the great work with Nuxt docs. Recently I've been trying to setup a custom nuxt loading component with and I found it hard to follow the docs because the flow on this particular was quite discontinuous on how to customize the the loading component. I saw a few issues on stack overflow where some people had this same problem despite the docs containing all the answers they need.

Hopefully this pull request makes the documentation for the loading component more elaborate and makes more Nuxt developers smile :D

### CODE REVIEW

**Comment by manniL** [comment]
Wow! Very well done :clap: 	
Thanks so much for the contribution!

**Comment by codebender828** [comment]
Thanks for maintaining this ❤️
